cloud/types.go

```diff
@@ -220,23 +220,24 @@ type (
 
 	// DeploymentReviewRequest is the review_request object.
 	DeploymentReviewRequest struct {
```

Comment:
Now that this is reused better just call it `ReviewRequest`. I was confused why this change here if the feature is not related to deployment.

Reply:
Fixed in https://github.com/terramate-io/terramate/pull/1451